multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -124,16 +124,7 @@ model:
     checkpoint_name: "yolov3_mobilenetv2_320_300e_coco"
     data_types:
       - "image"
```

Comment:
`image` -> `rois`?

Reply:
Image is the only modality for input so I think we shall keep datatype=image. We followed ner's implementation (where datatype is still text), using the problem type to decide which processor to use.

Reply:
I think `image` is better because the input data is still image for most mmlab applications.